frontend/lib/src/components/widgets/DataFrame/arrowUtils.ts

```diff
@@ -240,7 +240,19 @@ export function getColumnFromArrow(
   data: Quiver,
   columnPosition: number
 ): BaseColumnProps {
-  const title = data.columns[0][columnPosition]
+  const columnHeaders = data.columns.map(column => column[columnPosition])
+  const title = columnHeaders[columnHeaders.length - 1]
+
+  // If there are multiple header columns, join these names with a "/"
+  // and use it as group name, but ignore empty strings:
```

Comment:
Could you please extend this comment a little bit to provide an example? Also, the `.slice(0, -1)` part looks like the last columnHeader is treated differently. I guess this is the two level headers that we see in the screenshots above. So I believe the following is happening in the lines:
```javascript
let columnHeaders = ["a", "b", "c"]
group = "a / b"
```
And `"c"` is left out because it appears in the second-level headers correct?
I feel it'd make sense to add an example like this to the comment.

Reply:
Added more comment and example